README.md

```diff
@@ -76,6 +77,9 @@ const appProps: ApplicationProperties = {
     component: 'component',
 };
 
+/*
+Use this if you want to manually configure your transports for TCP/UDP or both
```

Comment:
"Use this if you want to manually configure your transports for TCP/UDP or both" -> they use it not only for tcp/udp configuration - mention it, it is a bit confusing.